README.md

```diff
@@ -61,19 +81,15 @@ $ cargo build [--release]
 
 ### Example Network Simulation
 
-An example is included to run a simulation of a network using serialization-serde ([https://serde.rs/](https://serde.rs/)) to efficiently serialize and deserialize Rust data structures.
+A basic example is included to run a network simulation.
 
 ```
-$ cargo run --example simulation --features=serialization-serde -- -h
+$ cargo run --example simulation -h
```

Comment:
I removed serialization-serde - unsure if any other flags are used at this point.

Reply:
That's correct. The flag `serialization-serde` has been removed. Wherever you see "--features=serialization-serde", please remove it.